core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/LocalSyncWriteModelContainer.java

```diff
@@ -103,10 +118,19 @@ void wrapForRecovery(final Runnable callable) {
   void commitAndClear() {
     wrapForRecovery(() -> {
       localWrites.commitAndClear();
-      configs.commitAndClear();
+      configWrites.commitAndClear();
       remoteWrites.commitAndClear();
     });
 
+    if (true) { // temporary: change this to a success indicator
```

Comment:
[nit] if you're planning on updating this in a future ticket of work, we should add the ticket number here. we've been trying to avoid ticket-less TODOs in the code where possible.

Reply:
no this has to change now, this is what is breaking those three tests that are still failing

Reply:
changed